public/scripts/prepare-create-assistant.js

```diff
@@ -0,0 +1,84 @@
+import { t } from './i18n.js';
+import { SlashCommandParser } from './slash-commands/SlashCommandParser.js';
+import { getContext } from './st-context.js';
+
+const ASSISTANT_NAME = 'Assistant';
+
+/**
+ * @desc Prepare the create assistant button.
+ * @returns {void}
+ */
+export function prepareCreateAssistant() {
+    const createAssistantBtn = document.querySelector('[data-which="create-assistant-btn"]');
+
+    if (createAssistantBtn) {
+        createAssistantBtn.addEventListener('click', async () => {
+            if (assistantAlreadyExists()) {
+                chatWithAssistant();
+            } else {
+                await createAssistant();
+                chatWithAssistant();
+            }
+        });
+    }
+}
+
+// region Helpers
+/**
+ * @desc Sleep for a given number of milliseconds.
+ * @param {number} ms
+ * @returns {Promise<void>}
+ */
+function sleep(ms) {
+    return new Promise(resolve => setTimeout(resolve, ms));
+}
+
+/**
+ * @desc Check if the assistant already exists.
+ * @returns {boolean}
+ */
+function assistantAlreadyExists() {
+    const { characters } = getContext();
+
+    return characters.some(character => character.name === ASSISTANT_NAME);
+}
+
+/**
+ * @desc Chat with the assistant.
+ * @returns {Promise<void>}
+ */
+async function chatWithAssistant() {
+    SlashCommandParser.commands.char.callback({ _scope: null, _abortController: null }, ASSISTANT_NAME);
+}
+
+/**
+ * @desc Create the assistant.
+ * @returns {Promise<void>}
+ */
+async function createAssistant() {
+    try {
+        const characterManagementButton = document.querySelector('#rightNavDrawerIcon');
+        const createCharacterButton = document.querySelector('#rm_button_create');
+        const characterNameInput = document.querySelector('#character_name_pole');
+        const firstMessageTextarea = document.querySelector('#firstmessage_textarea');
+        const saveCharacterButton = document.querySelector('#create_button_label');
+
+        characterManagementButton.click();
+        createCharacterButton.click();
```

Comment:
Tried if the actual function behind, `select_rm_create()` does work instead of having to click the UI?
If that works, it's always better than going the UI route.